### PR TITLE
Add a warning to not send SRs into the devel project

### PR DESCRIPTION
**Pull request by dcermak**

### CODE REVIEW

**Comment by dirkmueller** [comment]
this seems like a really awkward fix. you can set an attribute on the project to reject requests, which gives you the message rather than hoping for somebody to read your description (which noone does)

**Review by dirkmueller — CHANGES_REQUESTED** [review]
added it for you: https://build.opensuse.org/attribs/devel:BCI:Tumbleweed//OBS:RejectRequests/edit 

feel free to do that via the obs api instead.

File: src/staging/project_setup.py [hunk 1]
```diff
@@ -183,7 +183,11 @@ def generate_meta(
     title += f" for {os_version.full_os_name}"
 
     if project_type == ProjectType.DEVEL:
-        description = "BCI " + description
+        description = (
+            "BCI "
+            + description
+            + "\nThis project is automatically updated from git. Please do **not** send submit requests, send a pull request to the [bci-dockerfile-generator](https://github.com/SUSE/bci-dockerfile-generator) instead"
```
Comment (dirkmueller) [inline src/staging/project_setup.py:189]: asking for a PR might be too much. I think it is sufficient to ask for an issue to be filed

**Comment by dcermak** [comment]
> added it for you: https://build.opensuse.org/attribs/devel:BCI:Tumbleweed//OBS:RejectRequests/edit
> 
> feel free to do that via the obs api instead.

I've implemented that now

File: src/staging/bot.py [hunk 2]
```diff
@@ -1514,6 +1516,14 @@ async def configure_devel_bci_project(self) -> None:
         await self._run_cmd(f"""{self._osc} meta attribute {prj_name} -a OSRT:Config --set 'main-repo = standard
 pkglistgen-archs = ppc64le s390x aarch64 x86_64'""")
 
+        await self._run_cmd(
+            f"{self._osc} meta attribute {prj_name} -a OBS:RejectRequests"
+            " --set 'This project is automatically updated from git. Please do"
+            " **not** send submit requests. Either create an issue or send a "
+            "pull request to the [bci-dockerfile-generator]"
```
Comment (dirkmueller) [inline src/staging/bot.py:1523]: Does markdown work here?

**Review by dirkmueller — DISMISSED** [review]

File: src/staging/bot.py (hunk 2 above)
Comment (dcermak) [reply src/staging/bot.py:1523]: Not really: 
image

**Pull request merged**